dev/bk-merge-pr.py

```diff
@@ -211,6 +212,13 @@ def merge_pr(pr_num, target_ref, title, body, default_pr_reviewers, pr_repo_desc
 
     # The string "Closes #%s" string is required for GitHub to correctly close the PR
     close_line = "Closes #%s from %s" % (pr_num, pr_repo_desc)
+    # Find the github issues to close
+    github_issues = re.findall("%s #[0-9]{3,6}" % GITHUB_ISSUES_NAME, title)
+
+    if len(github_issues) != 0:
+        for issue_id in github_issues:
+            close_line += " and Closes #%s" % (issue_id)
```

Comment:
done, fixed this.

"This closes #<pr-num> from <pr-branch-name>, closes #<issue-id> and ..."